main.go

```diff
@@ -17,11 +18,25 @@ import (
 	"time"
 )
 
+var (
+	socksProxy *string
+	etcdPeers  *string
+)
+
 func main() {
-	var (
-		socksProxy = flag.String("socks-proxy", "", "Use specified SOCKS proxy (e.g. localhost:2323)")
-		etcdPeers  = flag.String("etcd-peers", "http://localhost:2379", "Comma-separated list of addresses of etcd endpoints to connect to")
-	)
+	app := cli.App("vulcan-config-builder", "Maps general service config to vulcan config.")
```

Comment:
Does this work? You need to wrap everything in a app.Action and do app.Run(os.Args) for the flags to be correctly parsed.